Sources/URLSession+Async.swift

```diff
@@ -31,41 +31,35 @@
 
 import Foundation
 
-extension URLSession {
+@available(iOS, deprecated: 15.0, message: "This method should only be called on iOS below 15.0")
+public extension URLSession {
```

Comment:
Best to keep these extension internal to the project and not part of the public API

```suggestion
extension URLSession {
```

Reply:
Good catch